src/main/java/mate/jdbc/models/Manufacturer.java

```diff
@@ -0,0 +1,51 @@
+package mate.jdbc.models;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer(String name, String country) {
+        this.name = name;
+        this.country = country;
+    }
+
+    public Manufacturer(Long id, String name, String country) {
+        this.id = id;
+        this.name = name;
+        this.country = country;
```

Comment:
```suggestion
        this(name, country);
```

It's better to use one constructor in another